index.html

```diff
@@ -1,312 +1,351 @@
+                  rows="10"></textarea>
               </li>
-            </ul> 
-            <button type="submit" class="formbutton">
+              
+            </ul>
+            <!--<button type="submit" class="formbutton">
+              Get in touch
+            </button>-->
+
+          </form>
+
+
+        </div>
+
+        <div class="f2">
+          <form action="https://formspree.io/f/xbjqpyeb" method="POST">
+            <ul class="form2">
+              <li>
+                <input type="text" required maxlength="30" placeholder="First name" name="First name">
+                <input type="text" required maxlength="30" placeholder="Last name" name="Last name">
+              </li>
+              <li>
+                <input type="email" required placeholder="Email address" name="user-email" id="emai">
+              </li>
+              <li>
+                <textarea class="txtarea" name="text" maxlength="500" required id="textare"
+                  placeholder="Hello TUFOIN, nice to meet you" cols="30" rows="10">
+                </textarea>
+              </li>
+              
+            </ul>
+            <button type="submit" class="formbutton bsecond">
               Get in touch
             </button>
           </form>
         </div>
-        <div class="online-icons">
-          <img class="line111" src="./image/longline.png" alt="long-line">
-          <ul class="icons2">
-            <li>
-              <a href="/"><i class="fab fa-github"></i></a>
-            </li>
-            <li>
-              <a href="/"><i class="fab fa-linkedin-in"></i></a>
-            </li>
-            <li>
-              <a href="/"><i class="fab fa-angellist"></i></a>
-            </li>
-            <li>
-              <a href="/"><i class="fab fa-twitter"></i></a>
-            </li>
-            <li>
-              <a href="/"><i class="fab fa-medium-m"></i></a>
-            </li>
-          </ul>
-          <img class="line22" src="./image/line.png" alt="short-line">
       </div>
-      </section>
-    </main>
-  </body>
-</html>
+      <div class="icons3">
+        <img src="./image/iconsssssb.png" alt="bottom-icon" >
+      </div>
+
+    </section>
+  </main>
+  <div class="bnav">
+    <div class="online-icons iconzz">
+      <img class="line111" src="./image/longline.png" alt="long-line">
+      <ul class="icons2">
+        <li>
+          <a href="/"><i class="fab fa-github"></i></a>
+        </li>
+        <li>
+          <a href="/"><i class="fab fa-linkedin-in"></i></a>
+        </li>
+        <li>
+          <a href="/"><i class="fab fa-angellist"></i></a>
+        </li>
+        <li>
+          <a href="/"><i class="fab fa-twitter"></i></a>
+        </li>
+        <li>
+          <a href="/"><i class="fab fa-medium-m"></i></a>
+        </li>
+      </ul>
+      <img class="line22" src="./image/line.png" alt="short-line">
+    </div>
+  </div>
+</body>
```

Comment:
- [ ] [OPTIONAL] It seems that you have a big empty space after your footer, let's fix this please 👍 
![image](https://user-images.githubusercontent.com/55465133/130270518-77113b8b-7caa-4a18-8e2f-eccedfd7d80e.png)